<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMenuBarMapEditor</class>
 <widget class="QMenuBar" name="WidgetMenuBarMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="nativeMenuBar">
   <bool>false</bool>
  </property>
  <widget class="QMenu" name="menuFloors">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="tearOffEnabled">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Floor</string>
   </property>
   <property name="toolTipsVisible">
    <bool>false</bool>
   </property>
   <addaction name="actionFloors"/>
   <addaction name="actionAutotile"/>
  </widget>
  <widget class="QMenu" name="menuEvents">
   <property name="title">
    <string>Object</string>
   </property>
   <addaction name="actionObject"/>
  </widget>
  <widget class="QMenu" name="menuFace_Sprite">
   <property name="title">
    <string>Face Sprite</string>
   </property>
   <addaction name="actionFace_Sprite"/>
   <addaction name="actionFix_Sprite"/>
   <addaction name="actionDouble_Sprite"/>
   <addaction name="actionQuadra_Sprite"/>
   <addaction name="actionWall_Sprite"/>
  </widget>
  <widget class="QMenu" name="menuMountain">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Mountain</string>
   </property>
   <addaction name="actionMountain"/>
  </widget>
  <widget class="QMenu" name="menu3D_object">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>3D object</string>
   </property>
   <addaction name="action3D_object"/>
  </widget>
  <widget class="QMenu" name="menuView">
   <property name="title">
    <string>View</string>
   </property>
   <addaction name="actionView"/>
  </widget>
  <action name="actionFloors">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/floor.png</normaloff>:/icons/Ressources/floor.png</iconset>
   </property>
   <property name="text">
    <string>Floor</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="text">
    <string>Objects</string>
   </property>
  </action>
  <action name="actionFace_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_face.png</normaloff>:/icons/Ressources/sprite_face.png</iconset>
   </property>
   <property name="text">
    <string>Face Sprite</string>
   </property>
  </action>
  <action name="actionFix_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_fix.png</normaloff>:/icons/Ressources/sprite_fix.png</iconset>
   </property>
   <property name="text">
    <string>Fix Sprite</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/pencil.png</normaloff>:/icons/Ressources/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
  </action>
  <action name="actionTin_of_paint">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/pin.png</normaloff>:/icons/Ressources/pin.png</iconset>
   </property>
   <property name="text">
    <string>Tin of paint</string>
   </property>
  </action>
  <action name="actionDouble_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_double.png</normaloff>:/icons/Ressources/sprite_double.png</iconset>
   </property>
   <property name="text">
    <string>Double Sprite</string>
   </property>
  </action>
  <action name="actionQuadra_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_quadra.png</normaloff>:/icons/Ressources/sprite_quadra.png</iconset>
   </property>
   <property name="text">
    <string>Quadra Sprite</string>
   </property>
  </action>
  <action name="actionWall_Sprite">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_walls.png</normaloff>:/icons/Ressources/sprite_walls.png</iconset>
   </property>
   <property name="text">
    <string>Wall Sprite</string>
   </property>
  </action>
  <action name="actiondfsfffffffffffffffffffffffff">
   <property name="text">
    <string>dfsfffffffffffffffffffffffff</string>
   </property>
  </action>
  <action name="actionAutotile">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/autotile.png</normaloff>:/icons/Ressources/autotile.png</iconset>
   </property>
   <property name="text">
    <string>Autotile</string>
   </property>
  </action>
  <action name="actionAnimated_Autotile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/autotile_animated_disabled.png</normaloff>:/icons/Ressources/autotile_animated_disabled.png</iconset>
   </property>
   <property name="text">
    <string>Animated Autotile</string>
   </property>
  </action>
  <action name="actionMountain">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/montain_disabled.png</normaloff>:/icons/Ressources/montain_disabled.png</iconset>
   </property>
   <property name="text">
    <string>Mountain</string>
   </property>
  </action>
  <action name="actionSlope">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/slope_disabled.png</normaloff>:/icons/Ressources/slope_disabled.png</iconset>
   </property>
   <property name="text">
    <string>Slope</string>
   </property>
  </action>
  <action name="action3D_object">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/object.png</normaloff>:/icons/Ressources/object.png</iconset>
   </property>
   <property name="text">
    <string>3D object</string>
   </property>
  </action>
  <action name="actionObject">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/ev.png</normaloff>:/icons/Ressources/ev.png</iconset>
   </property>
   <property name="text">
    <string>Object</string>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/view.png</normaloff>:/icons/Ressources/view.png</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <addaction name="menuFloors"/>
  <addaction name="menuFace_Sprite"/>
  <addaction name="menuMountain"/>
  <addaction name="menu3D_object"/>
  <addaction name="menuEvents"/>
  <addaction name="menuView"/>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
